app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -66,6 +66,11 @@ object FeatureFlags {
      */
     const val inactiveTabs = true
 
+    /**
+     * Enables cookie banner reduction setting in Nightly or debug builds.
```

Comment:
We should change all references of reduction like above and `preferences_cookie_banner_reduction` to handling or vice versa.